src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,11 +8,16 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
+    public static final String EXAMPLE = "123, some street, some unit, some postal code";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses should be in"
+            + "a/BLOCK, STREET, UNIT, POSTAL_CODE format";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
 
     public final String value;
+    private Block block;
```

Comment:
recollect the discussion during the tutorial on why you should not retain the `value` anymore.